### PR TITLE
gh-97669: Remove outdated example scripts

### PR DESCRIPTION
**Pull request by vstinner**
Remove outdated example scripts. Copy them from Python 3.11 if you still need them.

Removed scripts:

* byext.py
* byteyears.py
* cleanfuture.py
* copytime.py
* crlf.py
* db2pickle.py
* dutree.doc
* dutree.py
* find-uname.py
* find_recursionlimit.py
* finddiv.py
* findlinksto.py
* findnocoding.py
* fixcid.py
* fixdiv.py
* fixheader.py
* fixnotice.py
* fixps.py
* get-remote-certificate.py
* google.py
* highlight.py
* ifdef.py
* import_diagnostics.py
* lfcr.py
* linktree.py
* lll.py
* mailerdaemon.py
* make_ctype.py
* md5sum.py
* mkreal.py
* objgraph.py
* parse_html5_entities.py
* pdeps.py
* pickle2db.py
* pindent.py
* pysource.py
* reindent-rst.py
* rgrep.py
* suff.py
* texi2html.py
* untabify.py
* which.py

Changes:

* Remove test_fixcid, test_pdeps and test_pindent of test.test_tools.
* Remove get-remote-certificate.py changelog entry, since the script was removed.
* FYI there is a copy of crlf.py in Lib/test/test_lib2to3/data/.

* Issue: gh-97669

### CODE REVIEW

**Comment by vstinner** [comment]
Here is a first PR to remove scripts which look outdated to me. The remaining ones might be useful or are referenced in other parts of Python, so I prefer to write separated PRs for the other scripts that I plan to remove.

File: Tools/scripts/parse_html5_entities.py [hunk 1]
```diff
@@ -1,114 +0,0 @@
-#!/usr/bin/env python3
-"""
-Utility for parsing HTML5 entity definitions available from:
-
-    https://html.spec.whatwg.org/entities.json
-    https://html.spec.whatwg.org/multipage/named-characters.html
-
-The page now contains the following note:
-
-    "This list is static and will not be expanded or changed in the future."
```
Comment (ezio-melotti) [inline Tools/scripts/parse_html5_entities.py:10]: This script generates the list of HTML5 entities in `Lib/html/entities.py`.  In that file there are some comments there that reference this and that should be updated if this script is removed.

Initially the script was supposed to be run every once in a while to update the list, but the specs got recently updated with this note, so the only reason for keeping this script is to show how the list was generated (and possibly update it in the unlikely case that the upstream list changes).

FWIW I revised the script recently and removed the previous script (`Tools/scripts/parseentities.py`) in this PR:
* #92504

**Comment by vstinner** [comment]
Oh, I didn't want to remove md5sum.py in this PR. I also restored untabify.py since it's used by patchcheck.py.

File: Tools/scripts/parse_html5_entities.py (hunk 1 above)
Comment (vstinner) [reply Tools/scripts/parse_html5_entities.py:10]: I noticed that and I tried to keep it but it seems like I messed up with Git and I removed it. I fixed my PR.

**Comment by vstinner** [comment]
Ooops, I didn't plan to create a PR on the branch "main", so I created a new PR: PR #97675.

**Pull request closed without merging**